solr/core/src/test/org/apache/solr/search/TestExtendedDismaxParser.java

```diff
@@ -412,24 +413,20 @@ public void testFocusQueryParser() {
             nor);
 
     // throw in a numeric field
-    assertQ(req("defType","edismax", "mm","0", "q","Terminator: 100", "qf","movies_t foo_i"),
+    assertQ(req("defType","edismax", "mm","0", "q","Terminator: 100", "qf","movies_t foo_i","sow","true"),
             twor);
 
     assertQ(req("defType","edismax", "mm","100%", "q","Terminator: 100", "qf","movies_t foo_i", "sow","true"),
             nor);
-    // When sow=false, the per-field query structures differ (no "Terminator" query on integer field foo_i),
-    // so a dismax-per-field is constructed.  As a result, mm=100% is applied per-field instead of per-term;
-    // since there is only one term (100) required in the foo_i field's dismax, the query can match docs that
-    // only have the 100 term in the foo_i field, and don't necessarily have "Terminator" in any field.
```

Comment:
This comment basically describes my pending doubts, that I listed on Jira.
does mm=100%  in a field centric approach means all compatible query terms in at least a field?
If I use sow=false and mm=100% a document just having the numerical part is returned and I am not sure is correct